src/main/scala/io/opentargets/etl/preprocess/uniprot/DescriptionIdentifiers.scala

```diff
@@ -15,15 +15,15 @@ trait DescriptionIdentifiers {
   case class UniprotProcessedNames(recNames: Seq[String],
                                    altNames: Seq[String],
                                    symbols: Seq[String])
-  val prefixes
+  val descriptionPrefixes
     : List[String] = RECOMMENDED :: ALTERNATIVE :: ALTERNATIVE_ANTIGEN :: ALTERNATIVE_SHORT :: Nil
 
   def processNames(descriptions: Seq[String]): UniprotProcessedNames = {
     lazy val uniMap = descriptions
       .map(_.split("=").map(_.trim))
       .groupBy(_.head)
-      .filterKeys(k => prefixes.contains(k))
-      .mapValues(_.flatMap(_.drop(1).map(n => n.split("\\{").head.trim.stripSuffix(";"))))
+      .filterKeys(k => descriptionPrefixes.contains(k))
+      .mapValues(_.flatMap(_.drop(1).map(_.trim.stripSuffix(";").trim)))
```

Comment:
I think this is wrong. Consider the entry line:

```
DE   AltName: Full=Beta-N-acetylglucosaminidase {ECO:0000303|PubMed:11148210};
```
We only want `Beta-N-acetylglucosaminidase`, so we need to split on the `{` (regex escaped) to drop the metadata.

Reply:
I see you built this logic into `UniprotConverter.scala`; ignore.